site/content/contribute/gsoc/_index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Google Summer of Code 2021"
+description: "Ideas list for Google Summer of Code 2021"
+section: "contribute"
+---
+
+We are happy to announce that Mattermost is applying to [Google Summer of Code 2021](https://summerofcode.withgoogle.com) as an organization!
+
+For those of you who don’t know, Google Summer of Code is a program that connects students and open source organizations: After a selection process, students spend 10 weeks of their summer break contributing to an open source project, mentored by experienced contributors from the organizations while receiving a stipend from Google that helps them focus their summer time on the project.
+
+We want to make this experience successful for you, so we made sure to prepare a list of ideas that are interesting, have real-world application, require cross-functional collaboration, and are at the same time flexible in terms of output. In order to let you achieve your goals for the summer, we have an excellent team of people to help you during the whole process:
+
+-   Four software design engineers to mentor the projects:
+    -   Devin Binnie ([@devin.binnie](https://community.mattermost.com/core/messages/@devin.binnie))
+    -   Daniel Espino García ([@daniel.espino.garcia](https://community.mattermost.com/core/messages/@daniel.espino.garcia))
+    -   Alejandro García Montoro ([@alejandro.garcia](https://community.mattermost.com/core/messages/@alejandro.garcia))
+    -   Martin Kraft ([@martin.kraft](https://community.mattermost.com/core/messages/@martin.kraft))
+-   A product manager that will support the product decisions across all projects:
+    -   Eric Sethna ([@eric.sethna](https://community.mattermost.com/core/messages/@eric.sethna))
+-   The organization administrators:
+    -   Emily Cook ([@emily.cook](https://community.mattermost.com/core/messages/@emily.cook))
+    -   Alejandro García Montoro ([@alejandro.garcia](https://community.mattermost.com/core/messages/@alejandro.garcia))
+
+If you are [eligible to participate as a student](https://summerofcode.withgoogle.com/get-started/), take a look at the list of ideas below, and make sure to check if Mattermost is among the selected organizations on March 9!
+To keep track of the latest news from our side, make sure to join the [Google Summer of Code 2021 channel](https://community-daily.mattermost.com/core/channels/google-summer-of-code-2021), where we are centralizing all the communication related to the program.
+If we are finally selected as an organization, you will need to send your student proposals to [gsoc@mattermost.com](mailto:gsoc@mattermost.com). We will hopefully share more details in the coming weeks!
+
+Here’s a sneak peak of what’s coming next:
+
+-   March 9: Organizations announced
+-   March 29 - April 13: Student application period
+-   April 13 - May 17: Application review period
+-   May 17: Student projects announced
+-   May 17 - June 7: Community bonding
```

Comment:
Not entirely sure what "community bonding" means or implies here.

Reply:
It's how the GSoC site calls that period: it's the period where the student knows they're selected but before the coding period begins.